extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_dropdownLink.php

```diff
@@ -0,0 +1,5 @@
+<a href="<?= Sanitizer::encodeAttribute( $model['href'] ); ?>" class="wds-global-navigation__dropdown-link"
```

Comment:
I was referring to Logout link: https://github.com/Wikia/app/pull/11196/files#diff-4a1b823158dfc4984cc02216f2910d32R174